.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
What does this have to do with Angular CLI tools?

Reply:
The only relation this file has to Angular CLI tools is that it was generated by Angular CLI.

Otherwise, the reason why I let this file stay is because TSLint doesn't enforce the tab/spaces rule see [here](https://github.com/palantir/tslint/issues/2814). So we have to enforce the rule at the editor level.